fw/gfsm.h

```diff
@@ -104,6 +104,7 @@ enum {
 	/* Security rules enforcement. */
 	TFW_FSM_FRANG_REQ,
 	TFW_FSM_FRANG_RESP,
+	TFW_FSM_H2_HTTPS	= 11,
```

Comment:
I suggesting another solution, please consider it:
1. Postpone all initialization of http2/http1 proto after handshake completeness.
2. Add to `ss_proto_t` flag `negotiable` or similar.
3. By default always use `TFW_FSM_H2` or `TFW_FSM_HTTP/S ` proto type.
4. If `negotiable` is true, at ALPN matching stage you just change the proto, to specified by ALPN.

With this approach we don't need `TFW_FSM_H2_HTTPS` dummy state.